cirq-core/cirq/ops/clifford_gate.py

```diff
@@ -618,15 +618,15 @@ def from_unitary(u: np.ndarray) -> Optional['SingleQubitCliffordGate']:
             return None
         x = protocols.unitary(pauli_gates.X)
         z = protocols.unitary(pauli_gates.Z)
-        x_to = _to_pauli_transform(u @ x @ u.conj().T)
-        z_to = _to_pauli_transform(u @ z @ u.conj().T)
+        x_to = _to_pauli_tuple(u @ x @ u.conj().T)
+        z_to = _to_pauli_tuple(u @ z @ u.conj().T)
         if x_to is None or z_to is None:
             return None
         return SingleQubitCliffordGate.from_clifford_tableau(
             _to_clifford_tableau(x_to=x_to, z_to=z_to)
         )
 
-    def transform(self, pauli: Pauli) -> PauliTransform:
+    def transform(self, pauli: Pauli) -> Tuple[Pauli, bool]:
```

Comment:
this return type is a bit odd.  Should it be a dense pauli string?

Also is this a breaking change?

Probably should at least be documented.

Reply:
Making this into a DensePauliString introduces all sorts of circular dependencies, since DensePauliString depends on SingleQubitCliffordGate, and it is tricky to work around this.  I could do this if we have inline imports though.

Reply:
I changed this a bit.  I deprecated transform() but kept the return type the same.  I then added two functions (pauli_tuple()) which is the named tuple without the name, and dense_pauli_string which is the pauli_string).  What do you think?

Reply:
Much better.